src/pages/ProductList/ProductList.js

```diff
@@ -3,7 +3,185 @@ import './ProductList.scss';
+              </ul>
+
+              <ul className="productListLine">
+                <li className="eachProduct">
+                  <img
+                    className="eachProductPic"
+                    alt="productPicture"
+                    src="https://imageapac1.lacoste.com/dw/image/v2/BBCL_PRD/on/demandware.static/-/Sites-master/default/dw43a87826/PH731E-51N_031_20.jpg?imwidth=690&impolicy=product"
+                  />
+                  <div className="eachProductInfo">
+                    <span className="eachProductColor">+ 6 colours</span>
+                    <span className="eachProductName">
+                      (남성)프렌치 레귤러핏 반팔 폴로
+                    </span>
+                    <span className="eachProductPrice">139,000원</span>
+                  </div>
+                </li>
+                <li className="eachProduct">
+                  <img
+                    className="eachProductPic"
+                    alt="productPicture"
+                    src="https://imageapac1.lacoste.com/dw/image/v2/BBCL_PRD/on/demandware.static/-/Sites-master/default/dw43a87826/PH731E-51N_031_20.jpg?imwidth=690&impolicy=product"
+                  />
+                  <div className="eachProductInfo">
+                    <span className="eachProductColor">+ 6 colours</span>
+                    <span className="eachProductName">
+                      (남성)프렌치 레귤러핏 반팔 폴로
+                    </span>
+                    <span className="eachProductPrice">139,000원</span>
+                  </div>
+                </li>
+                <li className="eachProduct">
+                  <img
+                    className="eachProductPic"
+                    alt="productPicture"
+                    src="https://imageapac1.lacoste.com/dw/image/v2/BBCL_PRD/on/demandware.static/-/Sites-master/default/dw43a87826/PH731E-51N_031_20.jpg?imwidth=690&impolicy=product"
+                  />
+                  <div className="eachProductInfo">
+                    <span className="eachProductColor">+ 6 colours</span>
+                    <span className="eachProductName">
+                      (남성)프렌치 레귤러핏 반팔 폴로
+                    </span>
+                    <span className="eachProductPrice">139,000원</span>
+                  </div>
+                </li>
+                <li className="eachProduct">
+                  <img
+                    className="eachProductPic"
+                    alt="productPicture"
+                    src="https://imageapac1.lacoste.com/dw/image/v2/BBCL_PRD/on/demandware.static/-/Sites-master/default/dw43a87826/PH731E-51N_031_20.jpg?imwidth=690&impolicy=product"
+                  />
+                  <div className="eachProductInfo">
+                    <span className="eachProductColor">+ 6 colours</span>
+                    <span className="eachProductName">
+                      (남성)프렌치 레귤러핏 반팔 폴로
+                    </span>
+                    <span className="eachProductPrice">139,000원</span>
```

Comment:
```suggestion
                  <div className="productInfo">
                    <span className="colors">+ 6 colours</span>
                    <span className="productName">
                      (남성)프렌치 레귤러핏 반팔 폴로
                    </span>
                    <span className="price">139,000원</span>
```
scss에서 nesting을 해줄 수 있기 때문에 클래스명이 너무 과도하게 길어질 필요 없습니다